natgeo/mortar
# [feature/SYTO-819-2] Style the Story Card -- Video

https://jira.natgeo.com/browse/SYTO-819

To Test:
pull feature/SYTO-819-2 (I created a new branch because of all the merge conflict craziness earlier)
```
gulp serve
```
verify all tests pass and matches AC
